stable/mlflow/templates/database.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: postgresql.org/v1
```

This needs to be optional, so let's add a condition around it, something like this:

```suggestion
{{ if .Values.mflow.postgres.PostgresDatabaseEnabled -}}
apiVersion: postgresql.org/v1
```

---

Good point, thanks :)!